.env

```diff
@@ -1,6 +1,6 @@
 REACT_APP_TITLE = "ค่ายลานเกียร์ ครั้งที่ 21 - Dev"
 REACT_APP_DESCRIPTION = "ค้นหาความเป็น วิศวกร ด้วยมือของคุณเอง"
-REACT_APP_API_SERVER = "https://larngear20th-api.azurewebsites.net"
 REACT_APP_DEBUG = true
+REACT_APP_API_SERVER = "http://localhost:3000/api"

```

Comment:
อันนี้ยิงไปหา larngear.in.th/api ได้เลยไหมนะ

Reply:
อ้อหรือเพราะมี proxy แล้ว ?

Reply:
อันนี้เป็น Proxy ครับเดียวผมไปแก้ใน .env.production อีกที

Reply:
OK